src/SUMMARY.md

```diff
@@ -0,0 +1,15 @@
+# Summary
+
+- [Overview](./01-cosock.md)
+  - [Getting Started](./01.1-getting-started.md)
+- [Spawn](./02-spawn.md)
+- [Channels](./03-channels.md)
+- [Select](./04-select.md)
+- [Advanced: Run](./05-run.md)
+- [Advanced: Internals](./06-internals.md)
+- [Advanced: Integrating with Cosock](./07-setwaker.md)
+  - [Bounded Channel: Shared](./07.1-bounded-channel.md)
+  - [Bounded Channel: Recvr](./07.2-bounded-recvr.md)
+  - [Bounded Channel: Sendr](./07.3-bounded-sendr.md)
+  - [Bounded Channel: Finish](./07.4-bounded-final.md)
+- [Appendix A: Bounded Channel Limits](./Appendix-A.md)
```

Comment:
I would be nice to get some more well defined delineation beween the different kinds of content and make sure they're in  order of depth. (Quickstart -> User APIs -> Integrator APIs -> Internals). This is my first thought at reordering.

```suggestion
# Summary

- [Overview](./01-cosock.md)
  - [Installation](./01.1-installation.md)
  - [Quickstart](./01.2-quickstart.md)
- [Spawn](./02-spawn.md)
- [Channels](./03-channels.md)
- [Select](./04-select.md)
- [Advanced: Integrating with Cosock](./07-setwaker.md)
  - [Example: Bounded Channel](./07.1-bounded-channel.md)     [new: basic intro saying what you're about to go through]
    - [Bounded Channel: Shared](./07.1.2-bounded-shared.md)    [was: bounded-channel]
    - [Bounded Channel: Recvr](./07.1.3-bounded-recvr.md)
    - [Bounded Channel: Sendr](./07.1.4-bounded-sendr.md)
    - [Bounded Channel: Finish](./07.1.5-bounded-final.md)
    - [Bounded Channel Limitations](./07.1.6-bounded-limitations.md)  [was: Appendix-A]

- [Appendix A: Internals](./Appendix-A-internals.md)
- [Appendix B: Run](./Appendix-B-run.md)
```

Might even make sense to group 02, 03, & 04 into a section. :shrug:

Reply:
> [Quickstart](./01.2-quickstart.md)

I am curious what you feel would go into this chapter that isn't in `[Overview]`?

Reply:
I have updated the general structure to nearly what you have suggested here. Instead of putting the 2 internal details chapters into appendices I added an `Advanced` section with a foreword about how the content/audience for those has changed.